web/modules/custom/error_page/src/ErrorPageErrorHandler.php

```diff
@@ -0,0 +1,311 @@
+        // instead of an error message.
+        // @see debug()
+        if ($error['%type'] == 'User notice') {
+          $error['%type'] = 'Debug';
+          $class = 'status';
+        }
+
+        // Attempt to reduce verbosity by removing DRUPAL_ROOT from the file
+        // path in the message. This does not happen for (false) security.
+        if (\Drupal::hasService('app.root')) {
+          $root_length = strlen(\Drupal::root());
+          if (substr($error['%file'], 0, $root_length) == \Drupal::root()) {
+            $error['%file'] = substr($error['%file'], $root_length + 1);
+          }
+        }
+
+        // Require explicitly the renderer class, as the container might not be
+        // available yet and, as a consequence, the auto-loading might not work
+        // for extensions such as modules.
+        require_once __DIR__ . '/ErrorPageRenderer.php';
+        $markup = ErrorPageRenderer::render('message', static::$uuid, $original_exception);
+        $message = new FormattableMarkup($markup, $error);
+      }
+
+      if ($fatal) {
+        if ($is_installer) {
+          $message = 'The website encountered an unexpected error. Please try again later.' . '<br />' . $message;
+          // install_display_output() prints the output and ends script
+          // execution.
+          $output = [
+            '#title' => 'Error',
+            '#markup' => $message,
+          ];
+          install_display_output($output, $GLOBALS['install_state'], $response->headers->all());
+          exit;
+        }
+
+        if (!$original_exception) {
+          // Create one if we come from an error.
+          $original_exception = new \Exception($error['@message']);
+        }
+
+        // Require explicitly the renderer class, as the container might not be
+        // available yet and, as a consequence, the auto-loading might not work
+        // for extensions such as modules.
+        require_once __DIR__ . '/ErrorPageRenderer.php';
+        $markup = ErrorPageRenderer::render('page', static::$uuid, $original_exception);
+
+        $response->setContent($markup);
+        $response->setStatusCode(500, '500 Service unavailable (with message)');
+
+        $response->send();
+        // An exception must halt script execution.
+        exit;
+      }
+
+      if ($message) {
+        if (\Drupal::hasService('session')) {
+          // Message display is dependent on sessions being available.
+          \Drupal::messenger()->addMessage($message, $class, TRUE);
+        }
+        else {
+          print $message;
+        }
+      }
+    }
+  }
+
+  /**
+   * Returns an error message of a given type.
+   *
+   * @param string $type
+   *   The message type.
+   *
+   * @return string
+   *   The message text.
+   */
+  protected static function getMessage($type) {
+    $messages = [
+      FALSE => [
+        'standard' => '%type: @message in %function (line %line of %file).',
```

Comment:
Should we use constants for these message types? Also by removing the uuid message option we will have only one set of them.